Packs/DomainToolsIrisDetect/Integrations/DomainToolsIrisDetect/DomainToolsIrisDetect.yml

```diff
@@ -0,0 +1,1353 @@
+      description: The list of evidence supporting the threat scores.
+      type: Unknown
+    - contextPath: DomainToolsIrisDetect.Blocked.mx_exists
+      description: Indicates that there is no MX record for the domain.
+      type: Boolean
+    - contextPath: DomainToolsIrisDetect.Blocked.tld
+      description: The top-level domain.
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.id
+      description: The domain ID.
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.escalations.escalation_type
+      description: The type of escalation.
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.escalations.id
+      description: The escalation ID.
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.escalations.created
+      description: The date and time when the escalation was created.
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.escalations.created_by
+      description: The email address of the person who created the escalation.
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.monitor_ids
+      description: An array containing monitor IDs.
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.assigned_by
+      description: The email address of the person who assigned the domain to the watchlist.
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.assigned_date
+      description: The date and time when the domain was assigned to the watchlist (e.g.,"2023-04-20T13:13:23.000000+00:00").
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.registrant_contact_email
+      description: Registrant Email.
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.name_server
+      description: An array of objects containing name server information.
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.registrar
+      description: The domain registrar.
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.create_date
+      description: The date when the domain was created (e.g., "2023-04-21T01:56:14.652000+00:00").
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.ip.country_code
+      description: Country code for the ip.
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.ip.ip
+      description: Associated ip for the Domain.
+      type: String
+    - contextPath: DomainToolsIrisDetect.Blocked.ip.isp
+      description: Associated isp for the Domain.
+      type: String
+  - deprecated: false
+    description: This command will reset your fetch history.
+    execution: false
+    name: domaintools-iris-detect-reset-fetch-indicators
+  dockerimage: demisto/python3:3.10.12.63474
+  feed: false
+  isfetch: true
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
+tests:
+- No tests (auto formatted)
+fromversion: 6.9.0
```

Comment:
Please add here the default classifier, as we discussed in the demo with the key defaultclassifier. Here is an example:
defaultclassifier: CrowdStrike Falcon